_posts/2015-09-04-channel.html

```diff
@@ -3699,6 +3918,959 @@
+
+title=API+Testing+Playlist</code></pre>
+		<p class="note">The token in the above call is only an example.</p>
+
+		<h6>Positive response</h6>
+
+		<p>The response will contain the following headers:</p>
+		<table class="table responsive">
+			<tr>
+				<th>Header</th>
+				<th>Type</th>
+				<th width="100%">Meaning</th>
+			</tr>
+			<tr>
+				<td>Location</td>
+				<td>string</td>
+				<td>The URL of the new playlist.</td>
+			</tr>
+		</table>
+
+		<pre><code class="json">{
+	"playlist": {
+		"id": "111",
+		"title": "API Testing Playlist",
+		"total_duration": "176.256",
+		"created_at": 1484127917,
+		"updated_at": 1486569501,
+		"channel_id": 11223344
+	}
+}</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>invalid_token</td>
+				<td>401 Unauthorized</td>
+				<td>The provided Access Token is missing, revoked, expired or malformed</td>
+			</tr>
+			<tr>
+				<td>bad_request</td>
+				<td>400 Bad request</td>
+				<td>The playlist title is missing</td>
+			</tr>
+		</table>
+
+		<h3>Playlist details</h3>
+		<p>Get the details of a playlist</p>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">GET</td>
+				<td><a>https://api.ustream.tv/playlists/PLAYLIST_ID.FORMAT</a> (format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>A key-value structure under a "playlist" element. Example:</p>
+		<pre><code class="json">"playlist": {
+	"id": "111",
+	"title": "Title of a playlist",
+	"total_duration": "176.256",
+	"created_at": 1484127917,
+	"updated_at": 1486569501,
+	"channel_id": 11223344
+}</code></pre>
+
+		<p>Explanation of some values:</p>
+		<table class="table responsive">
+			<tr>
+				<th>key</th>
+				<th>type</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>total_duration</td>
+				<td>float</td>
+				<td>The sum of the duration of the videos on the playlist in seconds.</td>
+			</tr>
+		</table>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>invalid_token</td>
+				<td>401 Authorization required</td>
+				<td>If the playlist does not belong to a channel than the request must include a valid access token.</td>
+			</tr>
+			<tr>
+				<td>permission_denied</td>
+				<td>403 Permission denied</td>
+				<td>If the playlist does not belong to a channel than the authenticated user needs to have access to the playlist.</td>
+			</tr>
+			<tr>
+				<td>not_found</td>
+				<td>404 Not found</td>
+				<td>
+					404 can be the response in the following cases:
+					<ul>
+						<li>The playlist does not exist.</li>
+						<li>In case of channel playlist, the channel does not exists or the playlists on the channel page are not enabled.</li>
```

Comment:
Akkor viszont ez jó lehet:
"In case of a channel playlist and if either the channel does not exist or playlists on the channel page are not enabled." (további apró változtatások: "In case of channel playlist" -> "In case of a channel playlist", "does not exists" -> "does not exist", "or the playlists on the channel page" -> "or playlists on the channel page")